WebCssArchitecture.md

```diff
@@ -0,0 +1,78 @@
+CSS Architecture
+================
+
+Architectural principles
+------------------------
+
+- The UI-design principles(language) and CSS naming share a common structure
+- Mobile first, so the device with smallest screen has less css parsing
+- [Use immutable css objects](https://csswizardry.com/2015/03/immutable-css/)
```

Comment:
The document that is links to is a little confusing. Instead of including this, can we just summarise the key elements. From my understanding, it is being strict about using the BEM naming and not modifying class once they have been declared. Is that correct?